src/libraries/Microsoft.PowerFx.Core/Binding/BindInfo/FirstNameInfo.cs

```diff
@@ -36,6 +36,7 @@ internal sealed class FirstNameInfo : NameInfo
         // For FirstNames with BindKind.ThisItem, are we accessing a field of the control template instead of data?
         public readonly bool IsDataControlAccess;
         public readonly DName DataControlName;
+        public readonly bool IsPageable;
```

Comment:
>IsPageable [](http://example.com/codeflow?start=29&length=10)

My concern here is that this is taking "Paging" (a powerapps concept) and pushing it very deep into the the binder / symbols.